lib/wptool.js

```diff
@@ -258,7 +258,7 @@ function nativeEnumerate(wpsdk, options, next) {
 			} else {
 				var emulators = parseAppDeployCmdListing(out, wpsdk);
 				next(null, {
-					devices: [{name: 'Device', udid: 0, index: 0, wpsdk: null}],
+					devices: [],
```

Comment:
This is for the case when we call nativeEnumerate, which uses AppDeployCmd.exe, not wptool.exe. Right now we use this code path if the sdk version is 8.0 or 8.1, so in those cases we'd never report any devices.

I don't think that's right. I think in these cases we need to report this "fake" device. And if we want to use wptool for 8.1, then line 295 needs to change.